README.md

```diff
@@ -47,6 +47,10 @@ Stops mystem process. Will be automatically stopped on process.exit();
 
 Returns promise with lemmatized version for passed word
 
+### myStem.fetchAllGrammemes(word)
```

Comment:
It makes sense to name the call "extractAllGrammemes", as it is not fetching operation. What do you think?